src/libs/yup/config.ts

```diff
@@ -0,0 +1,9 @@
+import * as yup from "yup"
+
+yup.setLocale({
```

Comment:
Email error message will be shared across multiple schemas so it would a good idea to add this common error message for email here so you won't write the same error multiple times


yup.setLocale({
  mixed: {
    required: "forms:errors.requiredField",
  },
  string: {
    email: "forms:errors.notValidEmail",
  },
})

export default yup